docs/changelog.md

```diff
@@ -6,6 +6,20 @@ nav_order: 99
 
 # Version changelog
 
+ * **3.7.0**
+   * Adding `bcstats`, `correct`, and `groupdedup` to CLI.
```

Comment:
Let's be consistent if we end bullet points with periods or not

Reply:
Thanks. Updated - removed terminal .'s on all lines.